android/app/build.gradle

```diff
@@ -158,6 +158,8 @@ dependencies {
     implementation 'com.facebook.soloader:soloader:0.10.5+'
 
     implementation("androidx.swiperefreshlayout:swiperefreshlayout:1.0.0")
+    implementation project(':react-native-lottie-splash-screen')
+
```

Comment:
this line is not necessary 
[autolinking](https://github.com/react-native-community/cli/blob/main/docs/autolinking.md)